config.yml.sample

```diff
@@ -1,5 +1,6 @@
 development:
   cookie_secret: sample
+  cookie_ssl_only: false
```

Comment:
Ok, any other reasons? I'm not too familiar with the implications one way or the other.